bindings/python/docs/source/schemas.rst

```diff
@@ -0,0 +1,138 @@
+.. _schema usage:
+
+PyMongoArrow Schemas and Their Usage
+====================================
+
+While there are many similarities between `MongoDB Projections <https://www.mongodb.com/docs/manual/reference/operator/projection/positional/>`_
```

Comment:
I think it's a little confusing to start off this page immediately talking about projections. I would expect a page titled "PyMongoArrow Schemas and Their Usage" to start off with basic examples and then expand into different use cases. Could we title this page `Schema Examples` instead? That way we can expand it in the future to show examples for arrays and other data types.